operations/hardware_state_manager/Set_BMC_Management_Role.md

```diff
@@ -0,0 +1,80 @@
+# Set BMC Management Roles
+
+The ability to ignore non-compute nodes (NCNs) is turned off by default. Management nodes and NCNs are also not locked by
+default. The administrator must lock the NCNs and their BMCs to prevent unwanted actions from affecting these nodes. To more
+easily identify the BMCs that are associated with the management nodes, they need to be marked with the Management role in
+the Hardware State Manager (HSM), just like their associated nodes.
+
+This section only covers marking BMCs of management nodes with the Management role using HSM.
+For more information on locking or ignoring nodes, refer to the following sections:
+
+   * Hardware State Manager (HSM): See [Lock and Unlock Nodes](Lock_and_Unlock_Management_Nodes.md)
+   * Firmware Action Service (FAS): See [Ignore Node within FAS](../firmware/FAS_Admin_Procedures.md#ignore).
+   * Cray Advanced Platform Monitoring and Control (CAPMC): See [Ignore Nodes with CAPMC](../power_management/Ignore_Nodes_with_CAPMC.md)
+
+## Topics:
+
+   * [When To Set BMC Management Role](#when-to-set-bmc-management-role)
+   * [How To Set BMC Management Role](#how-to-set-bmc-management-role)
+
+
+<a name="when-to-set-bmc-management-role"></a>
+
+## When To Set BMC Management Role
+
+The BMCs of NCNs should be marked with the Management role as early as possible in the install/upgrade cycle, in order to prevent unintentionally taking down a critical node. The Management role on the BMCs cannot be set until after Kubernetes is running and the HSM service is operational.
```

Comment:
```suggestion
The BMCs of NCNs should be marked with the `Management` role as early as possible in the install/upgrade cycle to prevent unintentionally taking down a critical node. The `Management` role on the BMCs cannot be set until after Kubernetes is running and the HSM service is operational.
```